examples/1-Basic_Overview.ipynb

```diff
@@ -419,7 +419,7 @@
    "source": [
     "# asf-search queries CMR with page sizes of 500, \n",
     "# so setting maxResults=1000 means asf-search will have to query cmr twice, each time returning 500 products\n",
-    "large_results_generator = asf.search_generator(maxResults=1000, platform=asf.SENTINEL1A)\n",
+    "large_results_generator = asf.search_generator(maxResults=1000, platform=asf.PLATFORM.SENTINEL1A)\n",
```

Comment:
In light of the removal of the constants name-spacing that tried to sneak in, is this change actually intended as well?